site/content/entry/citra-progress-report-2018-Q1.md

```diff
@@ -0,0 +1,230 @@
+reuse the same data in the next frame, it has to go through the PCIe bottleneck again. But, we never
+use the same data again and hence gain a big performance boost.
+
+[wwylele](https://github.com/wwylele) tweaked the configuration UI and renamed some parts of it.
+However, this caused some confusion about the configuration window and GPU features.
+
+### What's changed in the UI ?
+
+With the initial release of GLvtx, we've had support for running shaders on the host GPU . So, we had
+a dropdown menu called **Shader Emulation** to select either CPU or GPU. The option has now been renamed
+as **Enable Hardware Shader**. When enabled, it makes use of the host GPU, and when disabled, it uses
+the CPU.
+
+Now as many of the users were using older hardware, we introduced an option called **Accurate Hardware
+Shader**. It was later renamed to **Accurate Multiplication**, because technically that's what it was
+doing. ***When enabled it would render games with more accuracy at the cost of slower performance***.
+Its the same with **Accurate Geometry Shader**. Earlier it was configurable via the ini file, but now
+its a check box to enable or disable accurate geometry shader.
+
+&nbsp;
+{{< sidebyside "image" "/images/entry/citra-progress-report-2018-q1/"
+    "renderer-settings.png=Old Graphics Tab"
+    "renderer-settings-new.png=New Graphics Tab" >}}
+
+During its time in Canary, a lot of testing was done by our contributors.They tested many number of
+games, under varied hardware conditions and submitted valuable telemetry data, crash logs, etc.,
+which helped us in fixing issues with the hardware shader support.
+
+**Note**:
+Unfortunately, due to driver issues with AMD GPUs and inadequate OpenGL support on macOS, the hardware
+shader might not perform better than the CPU renderer under these conditions. However, worry not, as
+research is being done into possible fixes that might come around sooner than you might think.
+
+Read more about hardware renderer improvements in detail [here](https://citra-emu.org/entry/improvements-to-hardware-renderer/).
+
+## Continuous Integration (CI) Improvements ([here](https://github.com/citra-emu/citra/pull/3647), [there](https://github.com/citra-emu/citra/pull/3649), [this](https://github.com/citra-emu/citra/pull/3706) and [that](https://github.com/citra-emu/citra/pull/374)) by [liushuyu](https://github.com/liushuyu)
+
+One of our major time and resource consuming parts of the project were our build generating environments.
+We use Travis and Appveyor to release fully functional builds for download. However, the time it took
+for a build to be made was very high and, if we stopped the building process abruptly, to do a quick
+fix, it took even more time to generate the build again. Thanks to [liushuyu](https://github.com/liushuyu)'s
+contributions, we were able to optimize this whole process and make it very efficient to easily build
+and release latest Citra Nightly and Canary releases.
```

Comment:
"release the releases"? "build and release the latest changes made to Citra" might sound better.